code/modules/mob/living/simple_animal/friendly/snek.dm

```diff
@@ -0,0 +1,22 @@
+/mob/living/simple_animal/friendly/snek
+	name = "Snake"
```

Comment:
Improper names should be in lowercase, so examines show `That's a snake` instead of `That's Snake`.